lesson13/counter.js

```diff
@@ -0,0 +1,9 @@
+export function setupCounter(element) {
+  let counter = 0
+  const setCounter = (count) => {
```

Comment:
Unnecessary files

Reply:
Same as the above comment, please see the latest commit 1117e22f84f53bb2e3620a6a3b3020db03ed2978